app/edge/devices/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface EdgeDeviceTableSettings {
+  hiddenColumns: string[];
+  autoRefreshRate: number;
+  pageSize: number;
+  sortBy: { id: string; desc: boolean };
+}
+
+export enum DeviceAction {
+  POWER_ON = 'power on',
```

Comment:
enums doesn't have to be CAPS_CASED, we know it's an enum from the type

Reply:
I replicated the case from `enum FeatureId`, which case do you think we should use?

Reply:
app/portainer/environments/types.ts

Reply:
just regular camelCase